mobile/src/main/java/com/gelakinetic/mtgfam/fragments/dialogs/SearchViewDialogFragment.java

```diff
@@ -64,7 +73,10 @@ public boolean onSelection(MaterialDialog dialog, View itemView, int which, Char
                             })
                             .positiveText(R.string.dialog_ok)
                             .build();
-                    return getParentSearchViewFragment().mFormatDialog;
+                    if (null != getParentSearchViewFragment()) {
```

Comment:
`getParentSearchViewFragment()` will never be null at this point, because if it was we would have already returned `DontShowDialog()` on line 56.

Reply:
`getParentSearchViewFragment()` will never be null at this point, because if it was we would have already returned `DontShowDialog()` on line 56.

Reply:
Android Studio was warning me about NPEs here still, so it got protection. My only theory is that something multithreaded is going on, and the return value can change between like 56 and here.

Reply:
I think this was your only comment, repeated a few times, so I'm only going to answer here. If it's